Microsoft.Toolkit.Uwp.UI/Extensions/ScrollViewer/ScrollViewerExtensions.Properties.cs

```diff
@@ -69,7 +69,7 @@ public static void SetHorizontalScrollBarMargin(FrameworkElement obj, Thickness
         }
 
         /// <summary>
-        /// Get <see cref="EnableMiddleClickScrollingProperty"/>. Returns `true` if middle click scrolling is enabled else retuen `false`
+        /// Get <see cref="EnableMiddleClickScrollingProperty"/>. Returns `true` if middle click scrolling is enabled else return `false`
```

Comment:
Side note - we should unify the way we display langwords in XML comments throughout the repository. Right now we have some displayed as plain text, some as inline markdown code (`true`), some as XML code blocks (`<c>true</c>`), some as XML langwords (`langword="true"`).

Reply:
We probably want a separate PR for that, I'll open an issue.

Reply:
Thanks @Sergio0694, great suggestion. I think it should be a separate issue, I've added #3441 for tracking based on your comment.